x-pack/legacy/plugins/apm/e2e/README.md

```diff
@@ -21,19 +21,18 @@ $ curl https://storage.googleapis.com/apm-ui-e2e-static-data/events.json --outpu
 
 ```shell
 $ cd x-pack/legacy/plugins/apm/e2e/cypress/ingest-data
-$ node replay.js --server-url http://localhost:8200 --secret-token abcd --events ./events.json
+$ node replay.js --server-url http://localhost:8201 --secret-token abcd --events ./events.json
 ```
->This process will take a few minutes to ingest all data
+
+> This process will take a few minutes to ingest all data
 
 4. Start Kibana
 
 ```shell
 $ yarn kbn bootstrap
-$ yarn start --no-base-path --csp.strict=false
+$ yarn start --no-base-path --config x-pack/legacy/plugins/apm/e2e/ci/kibana.e2e.yml
 ```
 
-> Content Security Policy (CSP) Settings: Your Kibana instance must have the `csp.strict: false`.
```

Comment:
This is now specified in `kibana.e2e.yml`. Not something the user has to worry about.